frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -63,49 +64,73 @@ const PageHeader = styled("div")(({ $variant, theme }: StyledVariantComponentPro
   width: "100%",
 }));
 
-const PageHeaderMainContainer = styled("div")({
+const PageHeaderBreadcrumbsWrapper = styled("div")(({ theme }: { theme: Theme }) => ({
+  marginBottom: theme.spacing(theme.clutch.spacing.xs),
+}));
+
+const PageHeaderMainContainer = styled("div")(({ theme }: { theme: Theme }) => ({
   display: "flex",
   alignItems: "center",
   height: "70px",
+  marginBottom: theme.spacing(theme.clutch.spacing.sm),
+}));
+
+const PageHeaderInformation = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "space-evenly",
+  height: "100%",
 });
 
-const Heading = styled(Typography)({
+const Title = styled(Typography)({
   lineHeight: 1,
 });
 
+const Subtitle = styled(Typography)(({ theme }: { theme: Theme }) => ({
+  color: alpha(theme.colors.neutral[900], 0.45),
+}));
+
 const WorkflowLayout = ({
   workflow,
-  variant = "standard",
-  heading = null,
+  variant,
+  title = null,
+  subtitle = null,
+  breadcrumbsOnly = false,
   hideHeader = false,
   children,
 }: React.PropsWithChildren<LayoutProps>) => {
+  const params = useParams();
   const location = useLocation();
+
+  if (variant === null) {
+    return <>{children}</>;
+  }
+
   const workflowPaths = workflow.routes.map(({ path }) => `/${workflow.path}/${path}`);
   const breadcrumbsEntries = generateBreadcrumbsEntries(
     location,
-    (url: string) =>
-      `/${workflow.path}` !== url &&
-      !workflowPaths.includes(url) &&
-      !workflowPaths.find(path => !!matchPath({ path }, url))
+    url => !!workflowPaths.find(path => !!matchPath({ path }, url))
```

Comment:
There's an issue with the list of workflow paths available:  

- When rendering the workflow paths not all possible routes are not available to this check, because some workflow s are fragmented between instances but not reconciled into one single workflow.